### PR TITLE
[ID-268] Set appID and orgID for notifications

### LINKED ISSUE #268: Set appID and orgID for notifications
Set appID and orgID in the env variabels for the multtenancy.

### PR DESCRIPTION
**Pull request by stefanvit**
## Description
Adding the enviorment variables for appID and orgID used for the NotificationBB dependances.

**Resolves #268 

## Review Time Estimate
Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors.
- [x] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
Set new env variables for appID and orgID

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [x] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [x] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [x] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

### CODE REVIEW

**Review by petyos — CHANGES_REQUESTED** [review]
Hi @stefanvit, please look the feedback I left in https://github.com/rokwire/lms-building-block/pull/60 Here the situation is identical.

**Review by petyos — DISMISSED** [review]
LGTM

**Review by petyos — APPROVED** [review]

**Review by mdryankov — DISMISSED** [review]
GTML

File: driven/notifications/adapter.go [hunk 1]
```diff
@@ -47,7 +49,7 @@ func (na *Adapter) SendNotification(recipients []Recipient, topic *string, title
 
 func (na *Adapter) sendNotification(recipients []Recipient, topic *string, title string, text string, data map[string]string) error {
 	if len(recipients) > 0 {
-		url := fmt.Sprintf("%s/api/int/message", na.baseURL)
+		url := fmt.Sprintf("%s/api/int/v2/message", na.baseURL)
```
Comment (mdryankov) [inline driven/notifications/adapter.go:52]: The V2 has different payload structure.

**Review by mdryankov — CHANGES_REQUESTED** [review]

**Pull request merged**